src/System.Net.Requests/tests/HttpWebRequestTest.cs

```diff
@@ -521,10 +1051,13 @@ public void Abort_BeginGetRequestStreamThenAbort_EndGetRequestStreamThrowsWebExc
 
             request.BeginGetResponse(new AsyncCallback(RequestStreamCallback), state);
 
-            request.Abort();
-            Assert.Equal(1, state.RequestStreamCallbackCallCount);
-            WebException wex = state.SavedRequestStreamException as WebException;
-            Assert.Equal(WebExceptionStatus.RequestCanceled, wex.Status);
+            if (!request.HaveResponse)
```

Comment:
See Remarks in https://msdn.microsoft.com/en-US/library/system.net.httpwebrequest.abort(v=vs.110).aspx

Is seems like Abort was always broken in netfx. Not sure if this is still true with the current corefx implementation.